packages/cli/cloud/src/login/command.ts

```diff
@@ -0,0 +1,16 @@
+import type { StrapiCloudCommand } from '../types';
+import { runAction } from '../utils/helpers';
+import action from './action';
+
+/**
+ * `$ cloud device flow login`
+ */
+const command: StrapiCloudCommand = ({ command, ctx }) => {
+  return command
+    .command('cloud:login')
+    .alias('login')
+    .description('Strapi Cloud Login')
```

Comment:
```suggestion
    .description('Strapi Cloud Login')
    .addHelpText('after', '\nAfter running this command, you will be prompted to enter your authentication information.')
```

Just a suggestion, since the login is currently only returning an error for me (I assume because cloud api isn't responding yet?) I got confused at first why there were no options and that I was doing something wrong, but `--help` didn't give me anything useful. This allows `strapi --help` to only display the "Strapi Cloud Login" and `strapi login --help` to display the extended message.

It might actually be nice to accept id and only prompt for password too though.